tests/composite_fk/models/tenant.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+
+class Tenant(models.Model):
+    pass
+
+
+class User(models.Model):
+    tenant = models.ForeignKey(Tenant, on_delete=models.CASCADE)
+
+    class Meta:
+        unique_together = ("tenant_id", "id")
```

Comment:
should we continue with unique_together or also consider constraints = [
            models.UniqueConstraint(....)] as well?

Reply:
it should work with both, we can add a test for `UniqueConstraint` too if that's what you're asking

Reply:
yes a test covering that part would be great

Reply:
will do, also need to test it with composite pk